### ISSUE TITLE
Recommendation for the batch size hyper-parameter and the number of epochs when training the GMMN generator

### ISSUE THREAD

**Issue by toannguyen1904** [valeoai/3DGenZ#11]
I see the gmmn generator takes quite a long time to train an epoch. Does that make sense?
If so, could you please recommend on the batch size and the number of epochs hyper-parameters?

**Comment by BjoernMichele** [comment]
Dear @toannguyen1904, 

we used the 20 epochs that are also in the default setting of the code for all semantic segmentation datasets. However, this does not mean they are optimal from a performance point of view, perhaps with less epochs you are good as well, or can achieve a better result with more/less epochs. 

However, taking the ZSL setting serious, it is hard to tune this as an hyperparameter, so we decided to keep it the same as used in [11] and not tune it. The batch size is 4 or 8 depending on the backbone (should also be the default in the code), and is taken from the batch size we used for the pre-training of the backbone. 

Best, 
Bjoern
